odoo/tests/common.py

```diff
@@ -758,6 +755,19 @@ def _spawn_chrome(self, cmd):
             return
         pid = os.fork()
         if pid != 0:
+            for _ in range(25):
+                time.sleep(0.1) # chrome takes some time to start up & open the port
+                listens = [cn.laddr for cn in psutil.Process(pid).connections() if cn.status == psutil.CONN_LISTEN]
+                if listens:
+                    break
+            else:
+                raise ChromeBrowserException('Failed to detect chrome devtools port after 2.5s.')
```

Comment:
Shouldn't we raise a SkipTest to avoid confusion for the developer ?
Just like in [_json_command](https://github.com/odoo/odoo/blob/22598c88638aaa2570c88f5bdf01d185abadd086/odoo/tests/common.py#L877) method ?